src/AspNetCoreModuleV2/InProcessRequestHandler/dllmain.cxx

```diff
@@ -103,31 +111,56 @@ __stdcall
 CreateApplication(
     _In_  IHttpServer        *pServer,
     _In_  IHttpContext       *pHttpContext,
-    _In_  PCWSTR             pwzExeLocation,
+    _In_  PCWSTR              pwzExeLocation,
     _Out_ IAPPLICATION      **ppApplication
 )
 {
-    HRESULT      hr = S_OK;
-    IAPPLICATION *pApplication = NULL;
-    REQUESTHANDLER_CONFIG *pConfig;
+    HRESULT                 hr = S_OK;
+    IN_PROCESS_APPLICATION *pApplication = NULL;
+    REQUESTHANDLER_CONFIG  *pConfig = NULL;
+
     // Initialze some global variables here
     InitializeGlobalConfiguration(pServer);
 
-    hr = REQUESTHANDLER_CONFIG::CreateRequestHandlerConfig(pServer, pHttpContext->GetApplication(), &pConfig);
-    if (FAILED(hr))
+    try
```

Comment:
if we want to user try-catch, we have to rescan all new operations like this to make sure they are safe.  I personally still prefer the NULL check.

Reply:
I'll do another scan now.